### PR TITLE
Implement feature flags v2 backend support

### LINKED ISSUE #7737: Implement Feature Flag v2 Backend Support
## Acceptance Criteria

- [x] Undo/revert all, or parts of, #7555 (TBD)
- [x] Create/seed (an empty) lookup table for feature flags
  - See example in prime-_router/src/main/resources/db/migration/V55__add_normalized_code_values_elr_metadata.sql_: `ALTER TABLE elr_result_metadata ADD specimen_source_site_code VARCHAR(512);`
- [x] Add "FeatureFlags" to organization object
  - This data is stored in setting table in DB and look for rows with type "ORGANIZATION". 
  - `Organization.kt` defines the object and will need to be updated and potentially and code that uses this object

### PR DESCRIPTION
**Pull request by arnejduranovic**
This PR reverts the changes made in https://github.com/CDCgov/prime-reportstream/pull/7555 and implements the "version 2" of feature flags as discussed in prior meetings and in the description of epic https://github.com/cdcgov/prime-reportstream/issues/7734. This new implementation relies on a new "virtual table" (aka lookup table) named rs_feature_flag to store feature flags and adds a featureFlags property to the Organization object to allow the assignment of feature flags to organizations. 

## Test Steps:
- Run FlyWay migration and confirm it deletes the two permission tables: `gradelw clean package`
- Manually create `rs_feature_flag` lookup table and verify DB looks correct:
  - `./prime lookuptables create -e local -n rs_feature_flag -i metadata/tables/local/rs_feature_flag.csv `
- Add a feature to an org
  - `./prime organization set -e local -i ~/Documents/assets/feature_org.yml `
  - Verify feature was added to org by reading back the org
    - `./prime organization get -e local -n feature_org `:
      
      ```
			name: "feature_org"
			description: "an org for testing the feature flags feature"
			jurisdiction: "FEDERAL"
			filters:
			- topic: "covid-19"
			  jurisdictionalFilter:
			  - "matches(ordering_facility_state, IG)"
			  qualityFilter: null
			  routingFilter: null
			  processingModeFilter: null
			- topic: "monkeypox"
			  jurisdictionalFilter:
			  - "matches(ordering_facility_state, IG)"
			  qualityFilter: null
			  routingFilter: null
			  processingModeFilter: null
			- topic: "full-elr"
			  jurisdictionalFilter:
			  - "%performerState = 'IG' or %patientState = 'IG'"
			  qualityFilter:
			  - "%hl7MessageType = 'R01'"
			  routingFilter: null
			  processingModeFilter: null
			featureFlags:
			- "message-tracker"
			version: 0
			createdBy: "local@test.com"
			createdAt: "2023-01-10T16:31:57.35Z"
      ```

## Changes
- Reverted changes in https://github.com/CDCgov/prime-reportstream/pull/7555
  - Implemented FlyWay migration to remove permission tables
- Created .csv to create initial feature flags
- Updated organization object to have featureFlags property

### Testing
- [x] Tested locally?
- [x] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #7737

## To Be Done
- Experience team needs to add "front-end" stories to https://github.com/cdcgov/prime-reportstream/issues/7734 to capture the work they will need to do to complete the feature flags feature (I added some hints of what needs to be done in the epic description).
- Once this is merged, I will need to manually run the FlyWay migration on Production and Staging DBs and initialize the featureFlags lookup table

### CODE REVIEW

**Comment by penny-lischer** [comment]
LGTM - will leave it up to @jimduff-usds for final approval.

**Comment by stephenkao** [comment]
Looks good to me as well!  Thanks for handling this!

File: prime-router/src/main/kotlin/Organization.kt [hunk 1]
```diff
@@ -12,9 +12,10 @@ open class Organization(
     val stateCode: String?,
     val countyName: String?,
     val filters: List<ReportStreamFilters>? = emptyList(), // one ReportStreamFilters obj per topic.
+    val featureFlags: List<String>? = emptyList(),
```
Comment (jimduff-usds) [inline prime-router/src/main/kotlin/Organization.kt:15]: Maybe add a brief comment here explaining what feature-flags are used for.

**Review by jimduff-usds — APPROVED** [review]
Looks great - thanks, @arnejduranovic !   I had one suggestion for a comment in the code, but otherwise all good.

**Pull request merged**